.circleci/config.yml

```diff
@@ -684,7 +699,9 @@ jobs:
           command: |
             ./script/create_release.sh ${CIRCLE_TAG} << pipeline.parameters.KUBEAPPS_REPO >>
   local_e2e_tests:
-    machine: true
+    machine:
+      image: << pipeline.parameters.DEFAULT_MACHINE_IMG >>
+    resource_class: << pipeline.parameters.DEFAULT_RESOURCE_CLASS >>
```

Comment:
In fact, this resource_class is different, as we only can choose from this list: https://circleci.com/docs/2.0/configuration-reference/#machine-executor-linux